concepts/50 React Components/70 TypeScript Support/10 Kinds of Types/10 Additional Event Parts.md

```diff
@@ -0,0 +1,19 @@
+**ItemInfo**
```

Comment:
It should be a sub-paragrpah of the Common Event Parts section, as well as EventInfo/NativeEventInfo.